Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -628,10 +632,29 @@ data Action = NoAction
             | NotifyDriverStatusCountDown Int String String String
             | UpdateProfileButtonAC PrimaryButtonController.Action 
             | SkipAccessibilityUpdateAC PrimaryButtonController.Action
-
+            | RepeatRide Int Trip
+            | Scroll Number
+            | WhereToClick 
+            | ShowMoreSuggestions 
+            | SuggestedDestinationClicked LocationListItemState
+            | RepeatRideCountDown Int String String String
+            | StopRepeatRideTimer 
+            | OpenLiveDashboard
+            | UpdatePeekHeight 
 
 eval :: Action -> HomeScreenState -> Eval Action ScreenOutput HomeScreenState
 
+eval ShowMoreSuggestions state = continue state { props {suggestionsListExpanded = not state.props.suggestionsListExpanded} }
+
+eval UpdatePeekHeight state = continue state{data{peekHeight = getPeekHeight state}}
+
+eval (Scroll item) state = do
+  let sheetState = if item > state.props.currSlideIndex then true
```

Comment:
@aryan-13 can we modify this as: 
`let sheetState = If item == state.props.currSlideIndex then state.props.isHomeScreenExpanded else item > state.props.currSlideIndex`